lib/generators/component/component_generator.rb

```diff
@@ -28,10 +28,34 @@ def create_locale_files
     end
   end
 
-  def append_frontend_packs
-    append_to_file "frontend/components/index.js" do
-      "import \"components/#{split_name.join("/")}/#{component_name}\";\n"
+  def import_to_packs
+    root_path = Pathname.new("frontend")
+    base_path = root_path + "components"
```

Comment:
Isn't it better to do `File.join(root_path, "components")`?

Reply:
https://ruby-doc.org/stdlib-1.9.3/libdoc/pathname/rdoc/Pathname.html#method-i-2B